### ISSUE TITLE
Anonymous account and several browser tabs

### ISSUE THREAD

**Issue by newsiberian** [reactioncommerce/reaction#656]
Hello,
when user interact with several tabs in one browser, we could see unexpected behavior of creation of anonymous user. Here is an illustration:
![tabs](https://cloud.githubusercontent.com/assets/11130487/12151257/0a1cf5fe-b4d0-11e5-8b7e-784afb7a9ed3.png)
Illustration is rough. I can not say exactly what sort of processes occur here.
Here is a video of this process:
![screencast-1452103572](https://cloud.githubusercontent.com/assets/11130487/12151513/540d7fa2-b4d1-11e5-996b-3c22cf104a9b.gif)

**Comment by newsiberian** [comment]
related [forums.meteor.com topic](https://forums.meteor.com/t/tracker-autorun-runs-copies-of-computation-in-several-browser-tabs/15978). Any help is appreciated)

**Comment by aaronjudd** [comment]
I'm not sure I understand the question, or the problem?

**Comment by newsiberian** [comment]
@aaronjudd could you reproduce this bug? This issue describes a bug. On a forum - a question about, as I understand, about the way to solve it. Maybe wrong one...

**Comment by zenweasel** [comment]
From @newsiberian Steps to reproduce:

1. Open more than 3 tabs with http:/localhost:3000
2. In one of these tabs log in as admin or register someone
3. Check all tabs. You should see that all tabs now logged in by user
4. Open developer tool in Recources tab (Crome) and look at LocalStorage. You should see you sessionId, userId
5. Now, in this log out from this user and look at changes in LocalStorage

**Comment by newsiberian** [comment]
Hello, I did some research with situation with tabs and I have some thoughts about it.

I did few tests with #705. In first test I turned off all subscriptions, except `shop` in `main.js`, all routings, except route `/` and leave only `core`, `core-theme`, `reaction-accounts` .

I've tested with 3 browsers. Two were with 4 tabs opened on `localhost:3000` in each and one with one tab. During the test I first log in as normal user, and then log out. Expected result: – `userId` should be change only one time.

In such configuration I had got the following results:
clean means: `core`, `core-theme`, `reaction-accounts` packages

configuration | successful attempts
------------------ | ---------------------------
clean | 9/10
clean + inventory package | 6/10
clean + shipping package | 7/10
clean + two analytics packages | 7/10
clean + social | 6/10
full stack | 5/10

Second test was with all turned off - default configuration, but with `Accounts.loginWithAnonymous()` was covered by setTimeout with different idle time:
![logout_bug1](https://cloud.githubusercontent.com/assets/11130487/12371538/c8854dd6-bc58-11e5-8f49-4133331db9ea.png)

idle time | successful attempts
----------- | ---------------------------
5000 | 10/10
4000 | 10/10
3000 | 1/10
2000 | 0/10
1000 | 0/10

Also I tried to move setTimeout inside check...
So, my thoughts. This is not reaction's bug. I think this is somehow connected with `Accounts` internal connection monitoring. It happens every 3 sec. [here](https://github.com/meteor/meteor/blob/master/packages/accounts-base/localstorage_token.js#L156)

I think it could looks like this:
![logout2](https://cloud.githubusercontent.com/assets/11130487/12371641/4a613958-bc5b-11e5-8934-c5cd5a8a74ca.png)

So, currently we have average result for 4 tabs - 7/10 of successful attempts, and 5/10 for 8 opened tabs.
 
I think some of you guys should think about creating of new issue in `meteor/meteor`
